addons/binding/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/protocol/SatelMessage.java

```diff
@@ -25,7 +25,7 @@
  * @since 1.7.0
  */
 public class SatelMessage {
-    private static final Logger logger = LoggerFactory.getLogger(SatelMessage.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(SatelMessage.class);
```

Comment:
I am not a big fan of making all static final members uppercase. I use upper case only for constants. But I'm OK with this change.

Reply:
Thanks for your quick response, this indeed a topic which splits the Java community :-)